### DISCUSSION TITLE
Is it possible to preload options of a getAttachFormRecordSelect ?

### DISCUSSION THREAD

**Discussion by otavio-araujo** [filamentphp/filament#2921] [Help] [answered]
Is it possible to preload options of a getAttachFormRecordSelect ?

**Comment by danharrin** [comment, marked as answer]
In v2.13>, you can call `->preloadRecordSelect()` on the `AttachAction` I think

**Reply by otavio-araujo** [reply]
Yep, it worked.

And, i tried with: ->options(Step::all()->pluck('nome', 'id')) and it worked too.

Thank You!
